core/domain/collection_jobs_one_off.py

```diff
@@ -79,9 +80,16 @@ def map(item):
                 'Update collection schema version to %d.' % (
                     feconf.CURRENT_COLLECTION_SCHEMA_VERSION))
             yield (
-                CollectionMigrationJob._MIGRATED_KEY,
+                CollectionMigrationOneOffJob._MIGRATED_KEY,
                 'Collection successfully migrated.')
 
     @staticmethod
     def reduce(key, values):
-        yield (key, values)
+        if key == CollectionMigrationOneOffJob._DELETED_KEY:
```

Comment:
Typically we use ast.literal_eval() to handle 'values' first, and then you don't need to do any further casting. Consider doing that instead?

(Ditto in other jobs, if applicable.)

Reply:
done.